### ISSUE TITLE
[Feature]: Time format in reminder list view

### ISSUE THREAD

**Issue by davebhc71** [uphy/obsidian-reminder#39]
### Goal or desired outcome of this feature

(1) 02:00 or 20:00 time display is not user friendly in Reminder Pane, and
(2) please allow for NOT showing the Time in Reminder Pane

### Describe the feature

Have options in the Plugin>Settings to accomplish (1) and (2) above.

**Comment by uphy** [comment]
> (1) 02:00 or 20:00 time display is not user friendly in Reminder Pane, and

Why `02:00`, `20:00` are not user friendly?

> please allow for NOT showing the Time in Reminder Pane

Why?

**Comment by davebhc71** [comment]
1. Why? Because 1/2 the world doesn’t use 20:00, but either 4:00am OR 4:00pm!

2. Many people DO NOT always need the hour or minute, the Day is enough!

That’s why it is a FEATURE REQUEST!

**Comment by uphy** [comment]
OK, I understand your request.

I will introduce new setting for time format of reminder list view someday.
Perhaps it will be momentjs date time format setting.

https://momentjs.com/docs/#/displaying/format/

**Comment by davebhc71** [comment]
EXACTLY! Every other Plug-in for Obsidian offers the “momentjs” options for date/time display! Much more ‘professional’ appearance.
